learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,96 @@
++ you have and know your **public IP** address
++ you have **a bitcoin and a litecoin address** for receiving mining reward
++ you may need to setup your network to allow other bitmark nodes to communicate with your bitmark node. You can check more information on "Important Notice On Network" below.
+
+## Installation
+
+### Step 0: Preparation
++ Have your **Public IP** ready
++ Make sure following ports are accessible from internet
+    + port	**2136** for connecting to other peer bitmarkd nodes
+    + port	**2135** for publishing blockchain events
+    + port 	**2130** for Bitmark node RPC server
+
+You can check more information on "Important Notice On Network" below.
+
+
+### Step 1: Download and Run Script
+
+Downloaded script will help you to manage bitmark node container. The  installation of the script will pull down a latest bitmark node container image from docker hub and then start a bitmark node container for you.  To check what functions the script supports and how to run it, simply run ``` bash install-node-linux-mac.sh```.  or ```bitmarkNode-HyperV.bat```.
+
+
++  Linux and mac users
+    + [download scripts for linux and mac](https://bitmark-node-docker-scripts.s3-ap-northeast-1.amazonaws.com/install-node-linux-mac.sh)
+    + execute bash install-node-linux-mac.sh [Your Public IP]. When run the script without providing [Your Public IP], the script will find your IP from Internet
+
+        ```bash install-node-linux-mac.sh 123.123.123.123```
+    + follow script instruction and select "1)Installation"
+
+    
++  Windows users
+    +  ensure that Hyper-V is turned on
+    +  [download script for windows](https://s3-ap-northeast-1.amazonaws.com/bitmark-node-docker-scripts/bitmarkNode-HyperV.bat)
+    + execute bitmarkNode-HyperV.bat [Your Public IP]. When run the script without providing [Your Public IP], the script find your IP from Internet
+
+        ``` bitmarkNode-HyperV.bat 117.166.111.123```
+    + follow script instruction and select "1)Installation"
++ execute **docker ps** in your command-line/shell to check if bitmark-node is running
+![docker ps result](https://i.imgur.com/l3dF4Hl.jpg)
+
+## Management Panel
+
+### Step 1: Open A Node-Management Webpage 
++ view our node-mangement webpage on http://127.0.0.1:9980 in a browser
++ input your BTC and LTC address for rewarding in a pop-up screen
+![Input your BTC&LTC Address](https://i.imgur.com/IRTlyjY.jpg?1)
+
+### Step 2: Run bitmarkd and recorderd
++ startup screen
+    + run bitmarkd by click "start button" of "Bitmark Node (bitmarkd)"
+    + after bitmarkd is running, run a recorderd by click "start button" of "Recorder Node (recorderd)"
+    +  if you don't want to use your machine to help us on mining, you can stop recorderd.
+
+![startup screen](https://i.imgur.com/aeONALb.jpg)
+
++ Running screen
+    + wait until 5 connections to establish a distribute network
```

Comment:
Do you mean `Wait until 5 nodes connected to establish a distributed network`?

Reply:
will change to Linzy's suggestion